analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -93,15 +94,17 @@ class AnalyzerResultBuilder {
      * Add the given [packageSet] to this builder. This function can be used for packages that have been obtained
      * independently of a [ProjectAnalyzerResult].
      */
-    fun addPackages(packageSet: Set<Package>) = apply {
+    fun addPackages(packageSet: Set<Package>) =
+        apply {
```

Comment:
I'm also wondering why this did not become a one-liner. Looks like the auto-formatting is pretty much broken...

Reply:
It seems that code blocks are not collapsed automatically (I also could not find a rule for that), but apart from that the other auto-formatting looks good to me.